samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -276,5 +353,13 @@ private TaskModuleResponse CreateTaskModuleResponse(string value)
                 },
             };
         }
+
+        private async Task<bool> SendActivityResponse(string responseText, ITurnContext turnContext, CancellationToken cancellationToken)
```

Comment:
Why SendActivityResponse here, when this updates the existing activity?
Also out of curiosity why return Task<bool> here instgead of Task-- looks like we only return true

Reply:
Ya I was trying to avoid having multiple HandleGraphCallThatMightNotBeFound, where errorHandler might or might not return a value. But made those changes.